test/rules/no-default-export/test.ts.lint

```diff
@@ -0,0 +1,4 @@
+export const default = 'VALID';
```

Comment:
do you think this should be valid? Under ES6 standards, this is actually the same as using a default export! (see https://github.com/Microsoft/TypeScript/issues/2242#issue-60179656). I'm leaning towards allowing this though